hibernate-core/src/main/java/org/hibernate/mapping/Table.java

```diff
@@ -50,7 +50,7 @@
 	/**
 	 * contains all columns, including the primary key
 	 */
-	private Map columns = new LinkedHashMap();
+	private Map<String, Column> columns = new LinkedHashMap<String, Column>();
```

Comment:
We don't need to duplicate generics spec on the right due to diamond operator. Other than that, this PR seems good. A good example we need to pay more attention to generics usage, partly because of legacy reason, partly because of coding sloppiness.

Reply:
I'd prefer diamond operator, but it breaks consistency with legacy code, It’s necessary to cleanup entire codebase in another commit.

Reply:
I agree with @NathanQingyangXu , better to use the diamond operator.

Reply:
done.